src/modules/peek/Peek.UI/MainWindow.xaml.cs

```diff
@@ -49,23 +48,33 @@ private void OnPeekHotkey()
         {
             if (AppWindow.IsVisible)
             {
-                this.Hide();
-                ViewModel.Files = new List<File>();
-                ViewModel.CurrentFile = null;
+                ExecuteHide();
```

Comment:
nit: I am not convinced with this naming. Maybe Init/Unit naming should be in here since this is the start of the app and init means starting the query and uninit means hiding+canceling+clearing.